test/sqladmin-fetcher.ts

```diff
@@ -71,6 +71,28 @@ const mockRequest = (
     });
 };
 
+t.test('constructor', async t => {
```

Comment:
These tests aren't really 'tested' as this suite may be refactored shortly; they only exist for coverage purposes.